ktor-http/ktor-http-cio/posix/src/io/ktor/http/cio/websocket/FrameNative.kt

```diff
@@ -26,7 +26,7 @@ actual sealed class Frame actual constructor(
      * (separated into several text frames so they have [fin] = false except the last one).
      * Note that usually there is no need to handle fragments unless you have a RAW web socket session.
      */
-    actual class Binary actual constructor(fin: Boolean, data: ByteArray) : Frame(fin, FrameType.BINARY, data) {
+    actual class Binary actual constructor(fin: Boolean, data: ByteArray) : Frame(fin, FrameType.BINARY, data, NonDisposableHandle) {
```

Comment:
Could you tell me the reason for the explicit argument here?

Reply:
Otherwise, it's an error: no value passed for parameter `disposableHandle`. It seems like some change happened in Kotlin with expect constructor default parameters, so that in the actual class no default arguments are available.